masu/processor/aws/aws_report_processor.py

```diff
@@ -227,10 +227,13 @@ def remove_temp_cur_files(self, report_path, manifest_id):
         removed_files = []
         for victim in victim_list:
             if victim['assemblyId'] != current_assembly_id:
-                LOG.info('Removing %s, completed processing on date %s',
-                         victim['file'], victim['completed_date'])
-                remove(victim['file'])
-                removed_files.append(victim['file'])
+                try:
+                    LOG.info('Removing %s, completed processing on date %s',
+                             victim['file'], victim['completed_date'])
+                    remove(victim['file'])
+                    removed_files.append(victim['file'])
+                except FileNotFoundError:
+                    LOG.info('Unable to locate file: %s', victim['file'])
```

Comment:
Do we want the log level to be warning or error on this?

Reply:
this should probably be warning